Pancakeswap-Agents/Large-LP-Token-Deposit-Withdraw/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "pancakeswap-large-lptoken-deposit-withdraw",
+  "version": "0.0.1",
+  "description": "Detects Large LP Token Deposits or Withdrawals on Pancakeswap",
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/PancakeSwap-Bots/Large-LP-Token-Deposit-Withdraw",
+  "chainIds": [
+    56
+  ],
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest",

```

```suggestion
    "test": "jest --detectOpenHandles",
```
Adding this because of the warning when running `npm test`